app/policies/issue_policy.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class IssuePolicy < ApplicationPolicy
+  def new?
+    !user.nil? && user.reporter?
+  end
+
+  def create?
+    !user.nil? && user.reporter?
```

Comment:
admin can create issue? I think idea of Policies is something what you can skip for now.
Any user could add issue. But only a few could moderate/delete them